diff --git a/build/Location.js b/build/Location.js
index fe8f479f650ea620bcdc5cf38abef6526c9d9054..39f18cf4981eb8301634ee783e3ca3afd935ed4f 100644
--- a/build/Location.js
+++ b/build/Location.js
@@ -1,4 +1,5 @@
-import { createPermissionHook, isRunningInExpoGo } from 'expo';
+import { createPermissionHook } from 'expo-modules-core';
+import { isRunningInExpoGo } from 'expo';
 import { Platform } from 'expo-modules-core';
 import ExpoLocation from './ExpoLocation';
 import { LocationAccuracy } from './Location.types';
diff --git a/build/index.js b/build/index.js
index 627c834ec8df86f017785339dd2f5e3ab58a5e39..7bb826d7f7606ffc63cf5d64ccd9e8558d4dcac0 100644
--- a/build/index.js
+++ b/build/index.js
@@ -1,4 +1,4 @@
-export { PermissionStatus } from 'expo';
+export { PermissionStatus } from 'expo-modules-core';
 export { LocationEventEmitter as EventEmitter } from './LocationEventEmitter';
 export { _getCurrentWatchId } from './LocationSubscribers';
 export { installWebGeolocationPolyfill } from './GeolocationPolyfill';
diff --git a/src/Location.ts b/src/Location.ts
index a7f84f721c9fccd526a11d0db0f1967b92ed21e4..a158a76559f90f3a779f96fda58b7eff99495778 100644
--- a/src/Location.ts
+++ b/src/Location.ts
@@ -1,4 +1,5 @@
-import { createPermissionHook, isRunningInExpoGo, type PermissionResponse } from 'expo';
+import { createPermissionHook, type PermissionResponse } from 'expo-modules-core';
+import { isRunningInExpoGo } from 'expo';
 import { Platform } from 'expo-modules-core';
 
 import ExpoLocation from './ExpoLocation';
