diff --git a/build/ExpoAudio.js b/build/ExpoAudio.js
index 977ef78d69ddfa720f7897ee8f6392be9f33d988..4ee5701749a19aed8afbcc6c805584795afc6d84 100644
--- a/build/ExpoAudio.js
+++ b/build/ExpoAudio.js
@@ -75,7 +75,7 @@ export function useAudioPlayer(source = null, options = {}) {
     const initialSource = useMemo(() => {
         return downloadFirst ? null : resolveSource(source);
     }, [JSON.stringify(source), downloadFirst]);
-    const player = useReleasingSharedObject(() => new AudioModule.AudioPlayer(initialSource, updateInterval, keepAudioSessionActive, preferredForwardBufferDuration), [
+    const player = useReleasingSharedObject(() => new AudioModule.AudioPlayer(initialSource, updateInterval, keepAudioSessionActive), [
         JSON.stringify(initialSource),
         updateInterval,
         keepAudioSessionActive,
@@ -330,7 +330,7 @@ export function createAudioPlaylist(options = {}) {
 export function createAudioPlayer(source = null, options = {}) {
     const { updateInterval = 500, downloadFirst = false, keepAudioSessionActive = false, preferredForwardBufferDuration = 0, } = options;
     const initialSource = downloadFirst ? null : resolveSource(source);
-    const player = new AudioModule.AudioPlayer(initialSource, updateInterval, keepAudioSessionActive, preferredForwardBufferDuration);
+    const player = new AudioModule.AudioPlayer(initialSource, updateInterval, keepAudioSessionActive);
     if (downloadFirst && source) {
         resolveSourceWithDownload(source)
             .then((resolved) => {
