diff --git a/lib/android/build.gradle b/lib/android/build.gradle
index a6e84de6f0d80babd7d512ddb54b163dec45a82f..4a8f8de272f14f43599498d633ba551c75df6e6b 100644
--- a/lib/android/build.gradle
+++ b/lib/android/build.gradle
@@ -5,30 +5,20 @@ def getExt(prop, fallback) {
     rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
 }
 
-buildscript {
-    ext.kotlin_version = '1.7.21'
-
-    repositories {
-        mavenCentral()
-    }
-
-    dependencies {
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-    }
-}
+def kotlin_version = getExt('kotlin', '2.1.20')
 
 android {
-    compileSdkVersion getExt('compileSdkVersion', 33)
-    buildToolsVersion getExt('buildToolsVersion', '33.0.0')
+    compileSdkVersion getExt('compileSdk', 36)
+    buildToolsVersion getExt('buildTools', '36.0.0')
 
     defaultConfig {
-        minSdkVersion getExt('minSdkVersion', 21)
-        targetSdkVersion getExt('targetSdkVersion', 33)
+        minSdkVersion getExt('minSdk', 24)
+        targetSdkVersion getExt('targetSdk', 36)
     }
 }
 
 dependencies {
     compileOnly 'com.facebook.react:react-native:+'
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     implementation 'com.amap.api:3dmap:9.6.0'
 }
